e2e/rook-test-framework/clients/rook_client.go

```diff
@@ -0,0 +1,102 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+	"github.com/rook/rook/e2e/rook-test-framework/enums"
+	"github.com/rook/rook/e2e/rook-test-framework/transport"
+)
+
+var (
+	STATUS_CMD  = []string{"rook", "status"}
+	VERSION_CMD = []string{"rook", "version"}
+	NODE_CMD    = []string{"rook", "node"}
```

Comment:
I agree about going to rook CLI, i think we were under the impression that the client library was somehow not exposed... We will go back and refactor this to use the lib, but not for this release...